platforms/forge/build.gradle.kts

```diff
@@ -12,42 +14,42 @@ architectury {
 dependencies {
     annotationProcessor("net.fabricmc:sponge-mixin:${Versions.Mod.mixin}")
     annotationProcessor("dev.architectury:architectury-loom:${Versions.Mod.architecuryLoom}")
-    
+
     shadedApi(project(":common:implementation:base"))
     "forgeRuntimeLibrary"(project(":common:implementation:base"))
-    
+
     implementation(project(path = ":platforms:mixin-common", configuration = "namedElements")) { isTransitive = false }
     "developmentForge"(project(path = ":platforms:mixin-common", configuration = "namedElements")) { isTransitive = false }
     shaded(project(path = ":platforms:mixin-common", configuration = "transformProductionForge")) { isTransitive = false }
-    
+
     forge(group = "net.minecraftforge", name = "forge", version = Versions.Forge.forge)
-    
+
     minecraft("com.mojang:minecraft:${Versions.Mod.minecraft}")
     mappings("net.fabricmc:yarn:${Versions.Mod.yarn}:v2")
-    
+
     //forge is not ok.
     compileOnly("org.burningwave:core:${Versions.Forge.burningwave}")
     "forgeRuntimeLibrary"("org.burningwave:core:${Versions.Forge.burningwave}")
 }
 
 loom {
     accessWidenerPath.set(project(":platforms:mixin-common").file("src/main/resources/terra.accesswidener"))
-    
+
     mixin {
         defaultRefmapName.set("terra.forge.refmap.json")
     }
-    
-    launches {
-        named("client") {
-            property("fabric.log.level", "debug")
-            property("mixin.env.disableRefMap", "true")
-        }
-        named("server") {
-            property("fabric.log.level", "debug")
-            property("mixin.env.disableRefMap", "true")
-        }
-    }
-    
+
+//    launches {
```

Comment:
Remove commented out code (it is deleted in all other subprojects)

Reply:
no, `property("mixin.env.disableRefMap", "true")` is needed for forgedev env. however launches stoped working after a dep update. it needs to be fixed but as forge is a unpublished and not working platform it will stay as a reminder for now